push_notifications/conf/app.py

```diff
@@ -0,0 +1,357 @@
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.six import string_types
+from .base import BaseConfig, check_apns_certificate
+from ..settings import PUSH_NOTIFICATIONS_SETTINGS as SETTINGS
+
+"""
```

Comment:
An example settings file in the source seems problematic since the user would need to provide api keys or certificates. I moved that massive docstring to the [wiki](https://github.com/jleclanche/django-push-notifications/wiki/Multiple-Application-Support) and expanded it.